io/post_fv3.F90

```diff
@@ -741,6 +744,9 @@ subroutine set_postvars_fv3(wrt_int_state,grid_id,mype,mpicomp)
           ncfrst(i,j) = 1.0
           bgroff(i,j) = spval
           rswinc(i,j) = spval
+          ltg1_max(i,j) = spval
+          ltg2_max(i,j) = spval
+          ltg3_max(i,j) = spval
```

Comment:
@SamuelTrahanNOAA I would think initializing ltg1_max, ltg2_max and ltg3_max might not be needed since they are already initialized in UPP code ALLOCATE_ALL.f , see [here](https://github.com/NOAA-EMC/UPP/blob/develop/sorc/ncep_post.fd/ALLOCATE_ALL.f#L424)

Reply:
@SamuelTrahanNOAA Those were set before we added all interval variable initializing in ALLOCATE.f.

Reply:
Those = spval are gone now